### PR TITLE
[trianglify]: fix trianglify types

### PR DESCRIPTION
**Pull request by fdaciuk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test trianglify`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/qrohlf/trianglify/blob/master/src/trianglify.js#L19-L33
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/trianglify/trianglify-tests.ts
1. types/trianglify/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 659c690c0bba94ec063d6fe8890b169e40de1649
</p>

**Comment by peterblazejewicz** [comment]
Let's wait a little more, the changes are braking, introducing new version and new module exports, etc.
ping @unindented

**Comment by fdaciuk** [comment]
@peterblazejewicz the types were from an older version of trianglify. The update is to use the current version of the lib (4.1.1), as you can see [here](https://github.com/qrohlf/trianglify)

File: types/trianglify/index.d.ts [hunk 1]
```diff
@@ -1,42 +1,100 @@
...
+        colorFunction?: ColorFunction;
         /** Width of stroke. Defaults to 1.51 to fix an issue with canvas antialiasing. */
-        stroke_width?: number | undefined;
-        /** An array of [x,y] coordinates to trianglulate. Defaults to undefined, and points are generated. */
-        points?: number[] | undefined;
+        strokeWidth?: number | undefined;
+        /** An array of [x,y] coordinates to triangulate. Defaults to `undefined`, and points are generated. */
+        points?: Points | undefined;
+        /** A boolean value to tell if trianglify should render the triangles or not. Defaults to `true` */
+        fill?: boolean | undefined;
```
Comment (peterblazejewicz) [inline types/trianglify/index.d.ts:42]: I think this is inconsistent, defaults like width/height should not have the defaults, so no undefined as supported type, that looks OK. But here (and other places)  `fill`, won't have default `null`, if someone passes `undefined:
https://github.com/qrohlf/trianglify/blob/master/src/trianglify.js#L48

Comment (fdaciuk) [reply types/trianglify/index.d.ts:42]: You're right, my bad 🙈. I'll test every property to make sure they are consistent and I'll update this PR =)

Thanks for your review ❤️

Comment (fdaciuk) [reply types/trianglify/index.d.ts]: Done @peterblazejewicz =)

**Review by peterblazejewicz — APPROVED** [review]
LGTM
@fdaciuk thanks!

**Comment by fdaciuk** [comment]
Ready to merge

**Pull request merged**